tests/ssh-hassh/test.yaml

```diff
@@ -4,7 +4,7 @@ features:
     - RUST
 
 args:
- - -k none --set stream.bypass=yes
+ - -k none --set stream.bypass=yes --set app-layer.protocols.ssh.encryption-handling=bypass
```

Comment:
Why do we need this change ?

Reply:
behavior change

- Encrypted SSH traffic bypass is now independently controlled through
  `app-layer.protocols.ssh.encryption-handling` setting. The setting can either
  be `bypass`, `default` or `full`. 
  To retain the previous behavior of encrypted traffic bypass
  combined with stream depth bypass, set
  `app-layer.protocols.ssh.encryption-handling` to `bypass` (while also
  setting `app-layer.protocols.tls.encryption-handling` to `bypass` and
  `stream.bypass` to `true`).

Previously SSH was bypassed after the session turned encrypted, now we need to be more explicit.

Reply:
Re 
> Previously SSH was bypassed after the session turned encrypted, now we need to be more explicit.

So previously when the SSH session turned encrypted and stream bypass was allowed then it bypassed the flow. This was not related to reaching the stream depth but only to the session state. 
Currently, you can replicate the behavior with setting encryption handling to bypass.